cleverhans/utils_tf.py

```diff
@@ -160,6 +161,10 @@ def model_train(sess, x, y, predictions, X_train, Y_train, save=False,
             nb_batches = int(math.ceil(float(len(X_train)) / args.batch_size))
             assert nb_batches * args.batch_size >= len(X_train)
 
+            # Indices to shuffle training set
+            index_shuf = list(range(len(X_train)))
+            shuffle(index_shuf)
```

Comment:
Can you provide a way of seeding this? Otherwise it will make the tutorial tests more random